modules/features/navigation/src/main/java/au/com/shiftyjelly/pocketcasts/navigation/FragmentTransactionHandler.kt

```diff
@@ -213,8 +213,7 @@ internal class FragmentTransactionHandler(
 /**
  * Info that gets serialized into the FragmentManager's fragment tag string
  */
-@Suppress("DataClassPrivateConstructor")
-internal data class TagStructure private constructor(
+internal class TagStructure private constructor(
```

Comment:
You can't do things like this. If it was a `data class` it had generated `equals()` and `hashCode()` and other parts of  code can rely on that.

Reply:
I've just removed to use the annotation instead 06f69b0f50baa0fb0e11b444413fc1703ef15701